docs/invitation_instructions.md

```diff
@@ -16,29 +16,10 @@ Have an invitation to connect to an Outline server? Follow these instructions to
 | [iOS ›](https://itunes.apple.com/app/outline-app/id1356177741) | Get Outline on the App Store |
 | [Windows ›](https://raw.githubusercontent.com/Jigsaw-Code/outline-releases/master/client/Outline-Client.exe) | Download Outline.exe and double click to launch. |
```

Comment:
@jyyi1 what are the S3 urls?

Reply:
Yeah, I think we should change to the S3 urls when we can.

Reply:
* Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe
* Linux: https://outline-releases.s3.amazonaws.com/client/Outline-Client.AppImage